Data/DataModel/Response.cs

```diff
@@ -52,6 +52,7 @@ public Response(Response source)
         /// The submitted text
         /// </summary>
         [Required]
+        [MaxLength(100)]

```

Comment:
Do you have data on this? This doesn't seem like an impossible bound to hit

Reply:
Adjusted per conversation